understanding/21/target-size-enhanced.html

```diff
@@ -7,87 +7,103 @@
-	  <ul>
-			  <li>the control is used frequently;</li>
-			  <li>the result of the interaction cannot be easily undone;</li>
-			  <li>the control is positioned where it will be difficult to reach, or is near the edge of the screen;</li>
-			  <li>the control is part of a sequential task.</li>
-	  </ul>
-		<section id="benefits">
-			<h3>Benefits</h3>
-			<ul>
-			  <li>Users who use a mobile device where touch screen is the primary mode of interaction</li>
-			  <li>Users with mobility impairments, such as hand tremors</li>
-			  <li>Users who find fine motor movements difficult</li>
-			  <li>Users who access a device using one hand</li>
-			  <li>Users with large fingers, or who are operating the device with only a part of their finger or knuckle</li>
-			  <li>Users who have low vision may better see the target</li>
-		  </ul>
-		</section>
+  	<section id="brief">
+  		<h2>In brief</h2>
+  		<dl>
+  			<dt>Goal</dt><dd>Controls can be operated more easily, especially on touch screens.</dd>
+  			<dt>What to do</dt><dd>Make custom targets at least 44 by 44 pixels.</dd>
+  			<dt>Why it's important</dt><dd>Some people cannot tap small objects.</dd>
+  		</dl>
+  	</section>
+  	<section id="intent">
+  		<h2>Intent</h2>
+  		<p>The intent of this success criterion is to help users who may have trouble activating a small target because of hand tremors, limited dexterity or other reasons. If the target is too small, it may be difficult to aim at the target. Mice and similar pointing devices can be hard to use for these users, and a larger target will help them greatly in having positive outcomes on the web page.</p>
+      <p>Touch is particularly problematic as it is an input mechanism with coarse precision. Users lack the same level of fine control as on inputs such as a mouse or stylus. A finger is larger than a mouse pointer, and generally obstructs the user's view of the precise location on the screen that is being touched/activated.</p>
+      <p>The issue can be further complicated for responsive/mobile sites which need to accommodate different types of fine and coarse inputs (e.g. a site that can be accessed both on a traditional desktop/laptop with a mouse, as well as on a tablet or mobile phone with a touch screen).</p>
+      <p>While this criterion defines a minimum target size, it is recommended that larger sizes are used to reduce the possibility of unintentional actions. This is particularly relevant if any of the following are true:</p>
+  	  <ul>
+        <li>the control is used frequently;</li>
+        <li>the result of the interaction cannot be easily undone;</li>
+        <li>the control is positioned where it will be difficult to reach, or is near the edge of the screen;</li>
+        <li>the control is part of a sequential task.</li>
+  	  </ul>
+      <p>The targets on a screen can have different purposes and uses, and this Success Criterion specifies how each is to be handled.</p>
```

Comment:
> and this Success Criterion specifies how each is to be handled.

This sounds decidedly odd. The understanding doc might but the normative text does not specify this in great detail (for good reasons).

Reply:
It's important to point out that this is the pre-existing content for the 2.1 understanding document, being restored. I'm not disagreeing that it is awkwardly written in places, but the intent of this PR is to restore the prior version.